src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-router.ts

```diff
@@ -13,4 +13,4 @@ JiraConnectEnterpriseAppRouter.post("/", JiraContextJwtTokenMiddleware, JiraConn
 JiraConnectEnterpriseAppRouter.route("/:uuid")
 	.get(csrfMiddleware, JiraJwtTokenMiddleware, JiraConnectEnterpriseAppCreateOrEdit)
 	.put(csrfMiddleware, JiraContextJwtTokenMiddleware, JiraConnectEnterpriseAppPut)
-	.delete(csrfMiddleware, JiraContextJwtTokenMiddleware, JiraConnectEnterpriseAppDelete);
\ No newline at end of file
+	.delete(JiraContextJwtTokenMiddleware, JiraConnectEnterpriseAppDelete);
```

Comment:
Not sure why we were using links here because we're not navigating outside the app 🤷 Had to wrap each list item in a div as well otherwise clicking on one would fire the other (both at the same time).